lib/generators/mobile_workflow/templates/model.rb.erb

```diff
@@ -11,8 +11,8 @@ class <%= class_name %> < <%= parent_class_name.classify %>
 <% end -%>
```

Comment:
shouldn't these .rb.erb also contain the magic comment?

Reply:
AH sorry I replied too quickly to your message.

Yes, sure! These files become `.rb` and they need the magic comment. This is why I added the module `RailsGeneratorFrozenStringLiteralPrepend` prepended to Thor:::Actions::CreateFile, and what this does is it adds the magic comment to all those files generated by the generator, such as user.rb, seeds.rb, ability.rb, controllers.rb, deserializers.rb, etc. So they do not raise the magic comment offense.